google-cloud-compute-v1/samples/acceptance/quickstart_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../quickstart"
+require_relative "helper"
+
+class ComputeQuickstartTest < Minitest::Test
+  def setup
+    @temp_instances = []
+  end
+
+  def teardown
+    client = ::Google::Cloud::Compute::V1::Instances::Rest::Client.new
+    operation_client = ::Google::Cloud::Compute::V1::ZoneOperations::Rest::Client.new
+    @temp_instances.each do |instance|
+      operation = client.delete project: project, zone: zone, instance: instance
+      if operation.status == :RUNNING
```

Comment:
I don't know if you want to wait_until_done here

Reply:
Good point, I think you actually don't need to wait here because it's just cleaning up.